nuxeo-arender-previewer-apb/defaults/main.yml

```diff
@@ -25,6 +25,7 @@ arender_previewer_monitoring_kibana_service: arender-monitor-kibana
 
 arender_previewer_monitoring_datadog_enabled: false
 arender_previewer_monitoring_datadog_key: ''
+arender_previewer_monitoring_datadog_tags: {}
```

Comment:
You need to add the same parameter in `apb.yml` if we want it to be configurable like here: [apb.yml](https://github.com/nuxeo/nuxeo-apb-catalog/blob/master/nuxeo-arender-rendition-apb/apb.yml#L91).

That means that it needs to be encoded as a string and afterwards converted as a JSON object in an intermediate object (in the `vars/main.yml`) file with the `from_json` ansible filter.